### PR TITLE
Fix spacing

### PR DESCRIPTION
**Pull request by DamienCassou**
Fix for #45

### CODE REVIEW

File: src/htmlCanvas.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ define(
 		 *		}
 		 *
 		 *
-		 * @param {string|jQuery|htmlTagBrush} [rootElement] Element to  "paint" on. If not supplied a document fragment will be created
+		 * @param {string|jQuery|htmlTagBrush} [rootElement] Element to	 "paint" on. If not supplied a document fragment will be created
```
Comment (BenjaminVanRyseghem) [inline src/htmlCanvas.js]: are you sure about the tab between `to` and ` "paint"`?

File: src/router/router.js [hunk 2]
```diff
@@ -519,7 +519,7 @@ define(
 
 			/**
 			 * Tries to resolve URL by matching the URL against all routes in
-			 * route table. Unless  `fallThrough`is set on the matched route router
+			 * route table. Unless	`fallThrough`is set on the matched route router
```
Comment (BenjaminVanRyseghem) [inline src/router/router.js]: why the tab here between `Unless` and `fallThrough`?

File: test/router/routerTest.js [hunk 3]
```diff
@@ -215,7 +215,7 @@ define([
 
 		it("resolveUrl pass optional value defaults to action", function(done) {
 			// Arrange a route that have two optional parameters
-			//  with defaukts
+			//	with defaukts
```
Comment (BenjaminVanRyseghem) [inline test/router/routerTest.js]: Why the tab between `//` and `with`?

**Comment by DamienCassou** [comment]
I took care of your comments.

**Comment by BenjaminVanRyseghem** [comment]
TL;DR;

**Comment by DamienCassou** [comment]
Why this tl;dr; ? Are you saying you didn't read the diff?
